src/workspaces/utils.ts

```diff
@@ -109,7 +109,7 @@ export interface WorkspacePolicy {
 
 export interface PolicyDescription {
   shortDescription: string;
-  longDescription: string;
+  longDescription?: string;
```

Comment:
We are moving away from showing info icons for the policies, so I didn't want to make up a long description for PHI tracking. I think that `longDescription` can be completely removed after WOR-1624 is done.